front/src/rangeofficer.js

```diff
@@ -250,9 +376,23 @@ const Tabletview = () => {
       <Typography
         variant="body1"
         align="center">
-        {tablet.Open[fin]}: {hours[0]} - {hours[1]}
+        {tablet.Open[fin]}:
+        &nbsp;
+        <Button
+          size="small"
+          variant="outlined"
+          style={{borderRadius:15}}
+          onClick={()=> setDialogOpen(true)}>
+          {hours.start} - {hours.end}
+        </Button>
       </Typography>
       &nbsp;
+
+      {dialogOpen ?
+       <TimePick tablet={tablet} fin={fin} scheduleId={scheduleId}
+                 hours={hours} setHours={setHours}
+                 dialogOpen={dialogOpen} setDialogOpen={setDialogOpen} />
+       : ""}
```

Comment:
Instead of sending an empty string here, should you send an empty element? (</> or <></>) , do they teach sending an empty string in Helsinki? Ultimately, this does not matter